wfl/Profiles/RestaurantCollectionProfile.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Hypermedia.Transforms;
+using Hypermedia;
+using Hypermedia.Affordances;
+using Symphono.Wfl.Controllers;
+using Symphono.Wfl.Models;
+using System;
+
+namespace Symphono.Wfl.Profiles
+{
+    public class RestaurantCollectionProfile : EntityProfile<RestaurantCollection>
+    {
+        protected override void Configure(EntityConfiguration<RestaurantCollection> configuration)
+        {
+            configuration
+                .UseRepresentationTransform(representation => representation
+                    .WithRepresentation("restaurant-collection")
+                    .WithRepresentation("collection")
+                )
+                .UseEnumerableEmbeddedSubEntityTransform(
+                    rc => rc.Restaurants,
+                    embedded => embedded
+                        .WithRelation("item")
+                        .WithConfiguration(embeddedConfiguration => embeddedConfiguration
+                            .UseProfile<Restaurant, RestaurantProfile>()
```

Comment:
It may have been specific to another project we were using Hypermedia for and never got ported over. The code you have is totally fine.